client/state/site-keyrings/selectors.js

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Returns true if we are requesting keyrings for the specified site ID, false otherwise.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether site keyrings is being requested
+ */
+export function isRequestingSiteKeyrings( state, siteId ) {
+	return get( state.siteKeyrings.requesting, [ siteId ], false );
+}
+
+/**
+ * Returns true if we are saving keyrings for the specified site ID, false otherwise.
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {Boolean}        Whether site keyrings is being requested
+ */
+export function isSavingSiteKeyrings( state, siteId ) {
+	return get( state.siteKeyrings.saveRequests, [ siteId, 'saving' ], false );
+}
+
+/**
+ * Returns the status of the last site keyrings save request
+ *
+ * @param  {Object}  state  Global state tree
+ * @param  {Number}  siteId Site ID
+ * @return {String}         The request status (peding, success or error)
+ */
+export function getSiteKeyringsSaveRequestStatus( state, siteId ) {
+	return get( state.siteKeyrings.saveRequests, [ siteId, 'status' ] );
```

Comment:
If no request is dispatched, we return here `undefined` ( `get()` default ), which is not `{String}` as JSdoc says, is that what it should be?